scripts/go_release.sh

```diff
@@ -0,0 +1,70 @@
+#!/bin/bash
+
+# Helper script to prepare a Go Release for the SDKs.
+
+set -e
+
+# Read the version number and build number from the respective files
+version_number=$(< internal/version.txt)
+build_number=$(< internal/version-build.txt)
+
+# Function to validate the version number format x.y.z(-beta.w)
+validate_version_number() {
+    local version="$1"
+    if [[ "${version}" =~ ^[0-9]+\.[0-9]+\.[0-9]+(-beta\.[0-9]+)?$ ]]; then
+        echo "Updated version number is: ${version}"
+        return 0
+    else
+        echo "Invalid version number format: ${version}"
+        return 1
+    fi
+}
+
+# Function to validate the build number format Mmmppbb
+validate_build_number() {
+    local build="$1"
+    if [[ "${build}" =~ ^[0-9]{1}[0-9]{2}[0-9]{2}[0-9]{2}$ ]]; then
+        echo "Updated build number is: ${build}"
+        return 0
+    else
+        echo "Invalid build number format: ${build}"
```

Comment:
pro-tip - you can ask the user directly to edit the version and build files in the script by running something like `${EDITOR:-nano} "$LOCATION_OF_FILE"

Reply:
Awesome, I changed it a bit to make sure the user inputs it correctly and not just error out.